client/css/style.css

```diff
@@ -1482,6 +1482,14 @@ background on hover (unless active) */
 	border-left: 1px solid var(--highlight-bg-color);
 }
 
+#chat .preview-size {
+	margin-left: 5px;
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
```

Comment:
Remove the verdor prefixes, if you rebase to master again, we have autoprefixer now.